tests/core/policies/test_ted_policy.py

```diff
@@ -712,14 +840,23 @@ class TestTEDPolicyWithRelativeAttentionMaxHistoryOne(TestTEDPolicy):
     max_history = 1
 
     def create_policy(
-        self, featurizer: Optional[TrackerFeaturizer], priority: int
+        self,
+        featurizer: Optional[TrackerFeaturizer],
+        priority: int,
+        model_storage: ModelStorage,
+        resource: Resource,
+        execution_context: ExecutionContext,
     ) -> Policy:
+        # TODO: seems like a duplicate of the test above??
```

Comment:
@dakshvar22 Is this duplicate on purpose?

Reply:
`max_history` is defined as `1` for this class whereas the above one will use the default `max_history=3` defined in `PolicyTestCollection`.

Reply:
oh dear. thanks! 🙈